src/main/java/org/jabref/gui/preferences/network/NetworkTabViewModel.java

```diff
@@ -179,6 +230,30 @@ private void storeProxySettings(ProxyPreferences newProxyPreferences) {
         proxyPreferences.setPassword(newProxyPreferences.getPassword());
     }
 
+    public void storeSSLSettings() {
+        trustStoreManager.flush();
+        sslPreferences.setUseCustomCertificates(customCertificatesUseProperty.getValue());
+        sslPreferences.setCustomCertificateVersion(customCertificateListProperty.stream()
+                                                                                .map(CustomCertificateViewModel::getVersion)
+                                                                                .collect(Collectors.toList()));
+        sslPreferences.setCustomCertificateThumbprint(customCertificateListProperty.stream().map(CustomCertificateViewModel::getThumbprint).collect(Collectors.toList()));
+        sslPreferences.setCustomCertificateValidFrom(customCertificateListProperty.stream()
+                                                                                  .map(CustomCertificateViewModel::getValidFrom)
+                                                                                  .map(this::localDateToEpochDayStr)
+                                                                                  .collect(Collectors.toList()));
+        sslPreferences.setCustomCertificateValidTo(customCertificateListProperty.stream()
+                                                                                .map(CustomCertificateViewModel::getValidTo)
+                                                                                .map(this::localDateToEpochDayStr)
+                                                                                .collect(Collectors.toList()));
+        sslPreferences.setCustomCertificateSerialNumber(customCertificateListProperty.stream().map(CustomCertificateViewModel::getSerialNumber).collect(Collectors.toList()));
+        sslPreferences.setCustomCertificateIssuer(customCertificateListProperty.stream().map(CustomCertificateViewModel::getIssuer).collect(Collectors.toList()));
+        sslPreferences.setCustomCertificateSigAlgorithm(customCertificateListProperty.stream().map(CustomCertificateViewModel::getSignatureAlgorithm).collect(Collectors.toList()));
+    }
+
+    private String localDateToEpochDayStr(LocalDate date) {
+        return String.valueOf(date.toEpochDay());
+    }
```

Comment:
refactoring to a separate method produces more boilerplate than inlining it. Or is there a different reason for extracting this?